core/tests/puppeteer-acceptance-tests/spec/voiceover-admin-tests/add-voiceover-artist-to-an-exploration.spec.ts

```diff
@@ -93,13 +104,17 @@ describe('Voiceover Admin', function () {
       explorationId =
         await explorationEditor.publishExplorationWithTitle('Exploration two');
 
-      await voiceoverAdm.navigateToExplorationEditor(explorationId);
-      await voiceoverAdm.navigateToExplorationSettingsTab();
+      await voiceoverAdmin.navigateToExplorationEditor(explorationId);
+      await voiceoverAdmin.navigateToExplorationSettingsTab();
 
-      await voiceoverAdm.expectVoiceoverArtistNotExists('voiceoverartist');
-      await voiceoverAdm.addVoiceoverArtistToExploration('voiceoverartist');
+      await voiceoverAdmin.expectToBeOmittedFromVoiceoverArtistList(
```

Comment:
For parity with expectVoiceoverArtistsListContains, let's call this expectVoiceoverArtistsListDoesNotContain

Reply:
Done.